### PR TITLE
Add a link to the get call api from the error page.

### PR DESCRIPTION
**Pull request by mrmeyers99**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-add-get-call-to-error-page.s3-website-us-east-1.amazonaws.com/

File: voice/errors.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ Errors that occur during an active call will be delivered to the `disconnectUrl`
 * User rate limits are reached
 * System rate limits are reached
 
+Errors can also be found for a call by [querying the call data in the api](methods/calls/getCallsCallId.md).
```
Comment (scapps-bw) [inline voice/errors.md]: api -> API

Comment (mrmeyers99) [inline voice/errors.md]:
```suggestion
Errors can also be found for a specific call by [querying the call data in the API](methods/calls/getCallsCallId.md). 
```

File: voice/errors.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ Errors that occur during an active call will be delivered to the `disconnectUrl`
 * User rate limits are reached
 * System rate limits are reached
 
+Errors can also be found for a specific call by [querying the call data in the API](methods/calls/getCallsCallId.md).
```
Comment (scapps-bw) [inline voice/errors.md]: The writer in me prefers the phrase "using the API" over "in the API." It's up to you if you agree or disagree, it probably doesn't matter that much. Let me know what you decide and I will approve.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-add-get-call-to-error-page.s3-website-us-east-1.amazonaws.com/

File: voice/errors.md (hunk 2 above)
Comment (mrmeyers99) [inline voice/errors.md]:
```suggestion
Errors can also be found for a specific call by [querying the call using the API](methods/calls/getCallsCallId.md). 
```

**Review by vsmith-bw — APPROVED** [review]

**Review by scapps-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-add-get-call-to-error-page.s3-website-us-east-1.amazonaws.com/

**Pull request merged**